docs/docusaurus/docs/cloud/deploy_gx_agent.md

```diff
@@ -0,0 +1,246 @@
+sidebar_label: 'Deploy the GX Agent'
+title: 'Deploy the GX Agent'
+id: deploy_gx_agent
+description: Deploy the GX Agent to use GX Cloud features and functionality.
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To use GX Cloud features and functionality, you need to deploy the GX Agent. The GX Agent is an intermediary between GX Cloud and your organization's data stores. GX Cloud does not connect directly to your data, all data access occurs within the GX Agent. GX Cloud sends jobs to the GX Agent, the GX Agent executes these jobs against your data, and then sends the job results to GX Cloud.
+
+The GX Agent is typically deployed in your organization's deployment environment, for example, in a development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows. To learn more about the GX Agent and deployment patterns, see [About GX Cloud](./about_gx.md).
+
+## Prerequisites
+
+- You have a [GX Cloud account](https://greatexpectations.io/cloud).
+
+## Get your access token and organization ID
+
+You need your access token and organization ID to deploy the GX Agent. Access tokens shouldn't be committed to version control software. 
+
+If you've used GX Cloud previously, you have your access token and organization ID, and you need to restart the GX Agent, see [Deploy the GX Agent](#deploy-the-gx-agent).
+
+<Tabs
+  groupId="copy-token"
+  defaultValue='new'
+  values={[
+  {label: 'New GX Cloud account', value:'new'},
+  {label: 'Existing GX Cloud account', value:'existing'},
+  ]}>
+<TabItem value="new">
+
+1. Sign in to GX Cloud.
+
+2. Complete the survey and then click **Continue to GX Cloud**.
+
+3. Copy and then paste the **Access token** and **Organization ID** values into a temporary file. You'll need them to deploy the GX Agent.
+
+4. Click **Deploy the GX Agent** and [deploy the GX Agent](#deploy-the-gx-agent).
+
+</TabItem>
+<TabItem value="existing">
+
+Use the information provided here to view your organization ID or create a new access token. This can be helpful if you've forgotten your organization ID or access token, and you need to restart the GX Agent.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **User access tokens** pane, click **Create user access token**.
+
+3. In the **Token name** field, enter a name for the token that will help you quickly identify it.
+
+4. Click **Create**.
+
+5. Copy and then paste the user access token into a temporary file. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your user access token and then save the file. 
+
+    GX recommends deleting the temporary file after you set the environment variables.
+
+8. [Deploy the GX Agent](#deploy-the-gx-agent).
```

Comment:
Same question for step 8 here as above.

Reply:
@annabarr, same answer! :) It's part of the process. Users need to know what to do after creating their access token and organization ID